tailnet/test/integration/suite.go

```diff
@@ -14,9 +16,29 @@ import (
 	"github.com/coder/coder/v2/testutil"
 )
 
+func sendRestart(t *testing.T, serverURL *url.URL, endpoint string) {
+	const reqTimeout = 2 * time.Second
+
+	serverURL, err := url.Parse(serverURL.String() + endpoint)
+	require.NoError(t, err)
+
+	client := http.Client{
+		Timeout: reqTimeout,
+	}
```

Comment:
To do timeouts we usually use contexts with timeouts. In tests you would probably do something like:

```go
ctx := testutil.Context(t, testutil.WaitLong)

req, err := http.NewRequestWithContext(ctx, http.MethodPost, ...)
if err != nil { ... }

resp, err := http.DefaultClient.Do(req)
```